internal/venice-common/src/test/java/com/linkedin/venice/utils/UtilsTest.java

```diff
@@ -183,4 +183,21 @@ public void testParseMap() {
   public void testSanitizingStringForLogger() {
     Assert.assertEquals(Utils.getSanitizedStringForLogger(".abc.123."), "_abc_123_");
   }
+
+  @Test
+  public void testParseCommaSeparatedStringMapFromString() {
```

Comment:
Let's rename this test along with the original function name since it is no longer a comma separated string map